array-manipulation-1/src/last.js

```diff
@@ -0,0 +1,8 @@
+/* exported last */
+// make for loop (i=0; i<array.length; i++)
+// return array[array.length-1];
+
+function last(array) {
+  for (var i = 0; i < array.length; i++);
```

Comment:
this loop is doing nothing for you. It doesn't even have a code block. It is just doing nothing over and over again for each item in the array.